assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -39,12 +44,84 @@ function DatePicker({ currentDate, onChange, onViewChange }) {
     [value, onChange]
   );
 
+  const updateTabIndexes = useCallback(
+    (setFocus = false) => {
+      // Set tabIndex to -1 for every except for the first button.
+      if (nodeRef.current) {
+        // Allow tabbing to sections inside the calendar.
+        const navButtons = [
+          ...nodeRef.current.querySelectorAll(
+            '.react-calendar__navigation button'
+          ),
+        ];
+        navButtons[0].tabIndex = '0';
+        navButtons.shift();
+        for (const btn of navButtons) {
+          btn.tabIndex = '-1';
+        }
+
+        // Dates / days.
+        const buttons = [
+          ...nodeRef.current.querySelectorAll(
+            '.react-calendar__viewContainer button'
+          ),
+        ];
+
+        let foundActive = false;
+        for (const btn of buttons) {
+          if (!btn.className.includes('react-calendar__tile--now')) {
+            btn.tabIndex = '-1';
+          } else {
+            btn.tabIndex = '0';
+            if (setFocus) {
+              // When changing view we need to explicitly set focus again,
+              // It seems to not be happening by default.
+              btn.focus();
+            }
+            foundActive = true;
+          }
+        }
```

Comment:
This works really great now. Just two things, the former being the more important one:

* If you select a date that is not today, close the date picker and open it again and tab to the date view, then you don't notice that "today" has focus, as it does not have a focus styling. We should really do something here - maybe even make it the same gray as a normally focused day or just a (much) lighter or darker yellow.
* What if two buttons have the `react-calendar__tile--now` class? It would technically leave two buttons with `tabIndex=0`, but I guess it's not possible?

Reply:
Added global style for adding a different color for the current date being focused, PTAL.

On having two buttons with the same class -- yes, that really shouldn't happen, can't see a scenario where two dates would be selected at the same time -- AFAIK the calendar doesn't support choosing more than one date. For now at least.